vllm/config.py

```diff
@@ -2324,6 +2334,70 @@ def _verify_args(self) -> Self:
 
         return self
 
+    @property
+    @deprecated(
+        "`num_redundant_experts` is deprecated and has been replaced with "
+        "`eplb_config.num_redundant_experts`. This will be removed in v0.12.0. "
+        "Please use `eplb_config.num_redundant_experts` instead.")
+    def num_redundant_experts(self) -> int:
+        return EPLBConfig.num_redundant_experts
```

Comment:
Another nit, this will always return the default value of `EPLBConfig`, not the instance value. (Same for the other deprecated fields).

```suggestion
        return self.eplb_config.num_redundant_experts
```

Reply:
Ah of course, because it's a property it doesn't have a value unless the class is instantiated. Even more reason to implement my recommended change!

Reply:
<img width="1840" height="350" alt="image" src="https://github.com/user-attachments/assets/d0882b85-2ced-4f78-9e2a-8f964033fbae" />

I change after, this issue still exist.